# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/time/nodef_rusage.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./resuse.h.orig	Fri May 31 20:15:43 2002
+++ ./resuse.h	Fri May 31 20:16:17 2002
@@ -39,6 +39,7 @@
 # if HAVE_WAIT3
 #  include <sys/resource.h>
 # else
+#  if 0  /* this is defined somewhere else */
 /* Process resource usage structure.  */
 struct rusage
 {
@@ -49,6 +50,7 @@
   ru_oublock, ru_msgsnd, ru_msgrcv, ru_nsignals,
   ru_nvcsw, ru_nivcsw;
 };
+#  endif
 # endif
 #endif
 
